### PR TITLE
[W3C API] Implement the device-radius media feature in CSS Media Queries

### PR DESCRIPTION
**Pull request by anawhj**
BUG=XWALK-4734

I would like to contribute the 'device-radius' media feature as one of the CSS Media Queries' new features, so I asked for two pull requests in chromium-crosswalk and blink-crosswalk. After all codes are merged and built well, it can be tested the command below.
xwalk http://anawhj.github.io/crosswalk-test-suite/webapi/webapi-rounddisplay-w3c-tests/ --round-display

Modified files in chromium-crosswalk
  - content/browser/renderer_host/render_process_host_impl.cc
  - content/browser/renderer_host/render_widget_host_impl.cc
  - content/public/common/content_switches.cc
  - content/public/common/content_switches.h 
  - content/common/view_messages.h
  - content/renderer/render_widget.cc

### CODE REVIEW

File: content/common/view_messages.h [hunk 1]
```diff
@@ -174,6 +174,7 @@ IPC_STRUCT_TRAITS_BEGIN(blink::WebScreenInfo)
   IPC_STRUCT_TRAITS_MEMBER(availableRect)
   IPC_STRUCT_TRAITS_MEMBER(orientationType)
   IPC_STRUCT_TRAITS_MEMBER(orientationAngle)
+  IPC_STRUCT_TRAITS_MEMBER(deviceRadius) //[device-radius]
```
Comment (joone) [inline content/common/view_messages.h]: https://code.google.com/p/chromium/codesearch#chromium/src/third_party/WebKit/public/platform/WebScreenInfo.h&sq=package:chromium&type=cs&l=39&rcl=1438687727

First, you need to apply the Blink change to https://github.com/crosswalk-project/blink-crosswalk.

Comment (anawhj) [inline content/common/view_messages.h]: Okay, I will progress the work in this repository after applying our blink changes to the blink-crosswalk repository.

**Comment by joone** [comment]
I think that we need to set the deviceRadius value by getting Android device information.

**Comment by anawhj** [comment]
I drop this thread and retry the progress.

**Pull request closed without merging**